modin/engines/dask/pandas_on_dask_delayed/axis_partition.py

```diff
@@ -40,27 +40,24 @@ def apply(
 
         if other_axis_partition is not None:
             return [
-                DaskRemotePartition(dask.delayed(obj))
-                for obj in deploy_func_between_two_axis_partitions(
+                DaskRemotePartition(obj)
```

Comment:
From here you should be able to just use the apply in `modin.engines.base.axis_partition.PandasOnXAxisPartition`